### PR TITLE
Add itk-meshtopolydata

### PR DESCRIPTION
**Pull request by thewtex**
This re-uses the PyPI binaries until we get the itk package built on
conda-forge.

@scopatz @ocefpaf @jakirkham I would like to make this package available on conda-forge so we can fully use the latest version of itk-jupyter-widgets with all conda-forge packages.

### CODE REVIEW

File: recipes/itk-meshtopolydata/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# install using pip from the whl files on PyPI
+
+POST=""
+PYPI_VER="${PY_VER:0:1}${PY_VER:2:3}"
+
+if [ `uname` == Darwin ]; then
```
Comment (ocefpaf) [inline recipes/itk-meshtopolydata/build.sh]: I know it is painful to write these in the recipe itself but we need to have some sort of checksum check and these if-clauses do not ensure we got the right tarball. Take a look at the [recipe for jaxlib](https://github.com/conda-forge/jaxlib-feedstock/blob/master/recipe/meta.yaml) for an example on how to use pre-processor selectors to do this in the recipe.

File: recipes/itk-meshtopolydata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - itk
+
+about:
+  home: http://github.com/InsightSoftwareConsortium/ITKMeshToPolyData
+  license: Apache 2.0
```
Comment (scopatz) [inline recipes/itk-meshtopolydata/meta.yaml]: This needs a license_file entry

File: recipes/itk-meshtopolydata/build.sh (hunk 1 above)
Comment (thewtex) [reply recipes/itk-meshtopolydata/build.sh]: Thanks for the pointer, Filipe! Done.

File: recipes/itk-meshtopolydata/meta.yaml (hunk 2 above)
Comment (thewtex) [reply recipes/itk-meshtopolydata/meta.yaml]: Done!

**Comment by ocefpaf** [comment]
@thewtex I fixed the wheels-jinja selection for you but seems that our linter is not ready for that much jinja. It seems to be working except for Linux+Python2.7, please check the error message in the CI and the comment in the recipe for more details.

**Comment by thewtex** [comment]
@ocefpaf thank you so much for your help! I am learning more conda recipe foo by example all the time...

Yes, your solution for the Linux+Python2.7 is correct -- we can just use the `mu` wheel. The [`u` is for Python compiler `--with-wide-unicode`](https://www.python.org/dev/peps/pep-3149/), which must be the default for conda-forge.

If the linter failure flag is a false positive, can we integrate?

**Comment by scopatz** [comment]
Yeah, let's ignore the linter for this.  Please open an issue on conda-smithy for this, though! Thanks!

**Comment by ocefpaf** [comment]
> Yes, your solution for the Linux+Python2.7 is correct -- we can just use the `mu` wheel. The [`u` is for Python compiler `--with-wide-unicode`](https://www.python.org/dev/peps/pep-3149/), which must be the default for conda-forge.
> 
> If the linter failure flag is a false positive, can we integrate?

@thewtex you may want to create another selector that adds the `mu` if you need the Linux+py27, I skipped it out of laziness.

**Comment by thewtex** [comment]
> Pull request successfully merged and closed 
Thanks!

>  Please open an issue on conda-smithy for this, though! 
Done here: https://github.com/conda-forge/conda-smithy/issues/1112

> @thewtex you may want to create another selector that adds the mu if you need the Linux+py27, I skipped it out of laziness.

@ocefpaf no problem-- Python 2.7 is not too important. Thanks again for your help!

**Pull request merged**